client/my-sites/preview/style.scss

```
@@ -1,4 +1,16 @@
 .main.preview {
 	height: 100%;
 	max-width: none;
+
+	.web-preview__toolbar {
+		position:sticky;
+		z-index:1;
+		top: var( --masterbar-height );
+	}
+}
+
+.layout.is-section-preview .layout__content {
+	// overflow cannot be set on an ancestor to .web-preview_toolbar otherwise that becomes the scrolling container
+	// rather than the viewport, and this will inhibit sticky behaviour. See https://github.com/w3c/csswg-drafts/issues/865
+	overflow:initial;
```

Nitpick: Ideally this should follow our CSS coding standards.
```suggestion
	overflow: initial;
```